### ISSUE TITLE
Docker run command is not working on Windows

### ISSUE THREAD

**Issue by akunaatrium** [dauxio/daux.io#40]
Either document that it is only for Linux or add Windows support.

**Comment by barjo** [comment]
Not sure if it's related but I have the following issue on linux:

```bash
docker run --rm -it -w /build -v "$PWD":/build daux/daux.io daux

Fatal error: Uncaught Error: Call to undefined method Todaymade\Daux\Console\Application::getTerminalDimensions() in /daux/libs/Console/Generate.php:46
Stack trace:
#0 /daux/vendor/symfony/console/Command/Command.php(252): Todaymade\Daux\Console\Generate->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /daux/vendor/symfony/console/Application.php(865): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /daux/vendor/symfony/console/Application.php(241): Symfony\Component\Console\Application->doRunCommand(Object(Todaymade\Daux\Console\Generate), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /daux/vendor/symfony/console/Application.php(143): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput) in /daux/libs/Console/Generate.php on line 46
```

It looks like this problem only happen with the latest docker image. It works fine with the `daux/daux.io:0.7.2` for me.

```bash
docker run --rm -it -w /build -v "$PWD":/build daux/daux.io:0.7.2 daux
Copying Static assets ...                                             [  OK  ]
...
```

**Comment by onigoetz** [comment]
@barjo Your issue has been fixed in 0.7.3
@akunaatrium  I'll need more details than just that. Daux works fine on windows, what is your setup, what is your issue ?

**Comment by onigoetz** [comment]
Closing because of no activity
